src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -73,6 +75,16 @@
   private static final String BATCH_SIZE_DISPLAY = "Batch Size";
   private static final int BATCH_SIZE_DEFAULT = 2000;
 
+  public static final String MAX_BULK_SIZE_BYTES_CONFIG = "max.bulk.size.bytes";
+  private static final String MAX_BULK_SIZE_BYTES_DOC =
+      "The maximum size (in bytes) of the bulk request to be processed by the Connector when"
+          + " writing records to Elasticsearch. The default is 10mb, the maximum setting possible."
+          + "If set to -1 or the condition for batch size by number of records is met first, this "
+          + "configuration will be ignored and the setting for batch size by number of records "
+          + "would be used instead.";
+  private static final long MAX_BULK_SIZE_BYTES_DEFAULT = 10 * MEGABYTES_TO_BYTES;
+  private static final String BULK_SIZE_BYTES_DISPLAY = "Max Bulk Size in Bytes";
```

Comment:
```suggestion
  private static final String BULK_SIZE_BYTES_DISPLAY = "Bulk Size (bytes)";
```

Reply:
default after display

Reply:
I think the default is already after the display. Do you mean before?